src/Application/Components/Store/StoreContainer.js

```diff
@@ -0,0 +1,29 @@
+import { connect } from 'react-redux';
+import Store from 'Store';
+import createActions from '../../Redux/actions';
+
+const visibleFilter = (assets) => {
+  return assets.filter((asset) => { return asset.unlocked; });
+};
```

Comment:
alternatively, i'd just put this in `mapStateToProps`.  Btw the bar's getting really low for what I think is important enough to even type, so buckle up I'm just going to stream of consciousness my own personal javascript style